DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -271,7 +271,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="19">
+  <class name="pat::PackedCandidate" ClassVersion="21">
+    <version ClassVersion="21" checksum="663492232"/>
```

Comment:
Yes, I saw the version 21 in 75X, so I closed this one temporarily so I could fix it.
I did not know about the version 22.  Thanks for letting me know.